installer/local_docker/tasks/main.yml

```diff
@@ -1,221 +1,7 @@
----
-- name: Manage AWX Container Images
-  block:
-    - name: Export Docker web image if it isnt local and there isnt a registry defined
-      docker_image:
-        name: "{{ awx_web_image }}"
-        tag: "{{ awx_version }}"
-        archive_path: "{{ awx_local_base_config_path|default('/tmp') }}/{{ awx_web_image }}_{{ awx_version }}.tar"
-      when: inventory_hostname != "localhost" and docker_registry is not defined
-      delegate_to: localhost
+- import_tasks: set_image.yml
 
-    - name: Export Docker task image if it isnt local and there isnt a registry defined
-      docker_image:
-        name: "{{ awx_task_image }}"
-        tag: "{{ awx_version }}"
-        archive_path: "{{ awx_local_base_config_path|default('/tmp') }}/{{ awx_task_image }}_{{ awx_version }}.tar"
-      when: inventory_hostname != "localhost" and docker_registry is not defined
-      delegate_to: localhost
+- import_tasks: standalone.yml
+  when: not use_docker_compose|default(False)
```

Comment:
This condition is met even when setting `use_docker_compose=false`.

It needs to become something like this:

```
when: not use_docker_compose | bool | default(False)
```

Reply:
After `| bool`, do I still need `| default(False)` ? My understanding is that bool will always output either `True`, `False` or `None` (which is `False`), so I can drop the default, no ?

If I needed a default different from the one `bool` does, it would rather be  `not use_docker_compose | default(False) | bool` ?

Reply:
(also, I'm not confortable relying on the prescedence of `|` over `not`, I'll add parentheses.)